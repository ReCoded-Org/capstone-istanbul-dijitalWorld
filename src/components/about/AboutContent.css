h2 {
  margin-bottom: 2rem;
  font-size: 2.8rem;
  color: #6c567c;
}

p {
  font-size: 1.5rem;
  color: #4a474d;
}
