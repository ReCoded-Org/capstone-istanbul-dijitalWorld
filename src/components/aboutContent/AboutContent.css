.articlesContainer {
  margin-top: 2rem;
}

.articlesContainer h2 {
  margin-bottom: 2rem;
  font-size: 3.3rem;
  color: #6c567c;
}

.articlesContainer p {
  font-size: 1.5rem;
  color: #4a474d;
}

.articleContainer {
  margin-top: 2rem;
  margin-bottom: 6rem;
}
