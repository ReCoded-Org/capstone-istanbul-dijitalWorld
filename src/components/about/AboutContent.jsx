import React from 'react';
import './AboutContent.css';
import { Col, Row, Container } from 'react-bootstrap';

const AboutContent = () => {

  return (
      <Container className="allContent">
        <Row>
          <Col>
          {/* <p>{props.card.res.map(function (abi, index) { return "Source # " + index + " " })}</p> */}

            <h2>Our Story</h2>
            <p>
              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
              incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud
              exercitation cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id
              est laborum.
            </p>
          </Col>
        </Row>
      </Container>
  );
};

export default AboutContent;
