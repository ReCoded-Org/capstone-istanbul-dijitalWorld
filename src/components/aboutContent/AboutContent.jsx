import React from 'react';
import './AboutContent.css';
import { useTranslation } from 'react-i18next';
import { Col, Row, Container } from 'react-bootstrap';

const AboutContent = () => {
  const { t } = useTranslation();
  return (
    <Container className="allContent">
      <Row>
        <Col>
          <div className="articleContainer">
            <h2>{t('about.header.0')}</h2>
            <p>{t('about.paragraph.0')}</p>
          </div>
          <div className="articleContainer">
            <h2>{t('about.header.1')}</h2>
            <p>{t('about.paragraph.1')}</p>
          </div>
          <div className="articleContainer">
            <h2>{t('about.header.2')}</h2>
            <p>{t('about.paragraph.2')}</p>
          </div>
        </Col>
      </Row>
    </Container>
  );
};

export default AboutContent;
