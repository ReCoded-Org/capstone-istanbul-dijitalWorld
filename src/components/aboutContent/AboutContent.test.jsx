import React from 'react';
import AboutContent from './AboutContent';
import renderer from 'react-test-renderer';

it('about content renders correctly', () => {
  const tree = renderer.create(<AboutContent />).toJSON();
  expect(tree).toMatchSnapshot();
});