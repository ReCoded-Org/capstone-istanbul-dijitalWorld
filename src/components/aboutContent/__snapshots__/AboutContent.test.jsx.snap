// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`about content renders correctly 1`] = `
<div
  className="allContent container"
>
  <div
    className="row"
  >
    <div
      className="col"
    >
      <div
        className="articleContainer"
      >
        <h2>
          about.header.0
        </h2>
        <p>
          about.paragraph.0
        </p>
      </div>
      <div
        className="articleContainer"
      >
        <h2>
          about.header.1
        </h2>
        <p>
          about.paragraph.1
        </p>
      </div>
      <div
        className="articleContainer"
      >
        <h2>
          about.header.2
        </h2>
        <p>
          about.paragraph.2
        </p>
      </div>
    </div>
  </div>
</div>
`;
